build/changelog.py

```diff
@@ -36,11 +72,11 @@ def get_subject(commit_message):
     return commit_message.splitlines()[0]
 
 
-def get_changelog_message(commit_message, repo_url):
+def get_changelog_message(commit_message, author, repo_url):
```

Comment:
Nit: Similar to above, the name `author` here does not communicate what this variable holds. I'd use something like `mention` here.